UKHomeOffice/nmsw-ui
# NMSW-138 - Upgrade to node 18, bump app version

# Ticket

NMSW-138

----

## AC

Upgrade to Node 18

----

## To test

- Delete your `/node-modules` folder and everything within
- Delete your `package-lock.json` file
- Run `npm i`

- Run `npm start` 
- > app should run on localhost:3000

- Run `npm run test`
- > tests should pass

- Stop the app
- Run `docker build -t nmsw-ui .` to build the app Docker container
- Run `docker run --name nmswui -p 8080:8080 nmsw-ui .` to run the Docker container
- > app should run on localhost:8080


----

## Notes
